### DISCUSSION TITLE
How about making braces less visible in default theme?

### DISCUSSION THREAD

**Discussion by ssbarnea** [Textualize/rich#895] [Q&A]
The default theme makes emphasizes braces in code instead of making them dimmer.

While I know how to tune the theme myself, I would find useful to have a defaults that do not need tuning as this would make the experience more consistent between various tools (current defaults invites to add custom theme)

I am not sure what was the reason for this as I often see the use-case for the opposite, regardless if we talk about plain text or code snippets.

* Most code editors do the same: display parentheses different than normal text but in a dimmed way
* they are less important than the content

Example using default theme, which exemplifies how a dictionary becomes harder to read due to bold/highlight and also where text commented and even `plural(s)` become harder to read.

![](https://sbarnea.com/ss/Screen-Shot-2021-01-08-15-37-03.48.png)

Example of vscode where parantheses are dimmed:
![](https://sbarnea.com/ss/Screen-Shot-2021-01-08-15-40-39.85.png)

IMHO, a default like  `"repr.brace": Style(bold=False, dim=True),` would improve the visual experience.

**Comment by willmcgugan** [comment]
I find bold braces help to delimit the data. Without them its hard to see where one data structure begins and the next ends.

I would only change the default if there was a clear consensus about it.

**Reply by ssbarnea** [reply]
Lets see if there is some agreement on that.  There is no cost on letting this question linger for months.

Usually when I have nested data where level is important, I format it. I doubt any color/style can replace a multiline indentation for those.
